uat/mqtt-client-control/src/main/java/com/aws/greengrass/testing/mqtt/client/control/implementation/EngineControlImpl.java

```diff
@@ -207,7 +218,7 @@ public void onMqttDisconnect(String agentId, int connectionId, Mqtt5Disconnect d
     private void unregisterAllAgent() {
         agents.forEach((agentId, agentControl) -> {
             if (agents.remove(agentId, agentControl)) {
-                agentControl.stopAgent();
+                // agentControl.stopAgent();
```

Comment:
should this be commented out or removed?

Reply:
fixed